SpatialGDK/Source/SpatialGDK/Public/Interop/SpatialDispatcher.h

```diff
@@ -31,10 +30,38 @@ class SPATIALGDK_API USpatialDispatcher : public UObject
 	void Init(USpatialNetDriver* NetDriver);
 	void ProcessOps(Worker_OpList* OpList);
 
+	// AddOpCallback returns a callback ID which is incremented on each callback that is registered.
+	// ComponentId must be in the range 1000 - 2000.
+	// Callbacks can be deregistered through passing the corresponding callback ID to the RemoveOpCallback function.
+	template<typename T>
+	using UserCallback = TFunction<void(T)>;
```

Comment:
We should probably make this a function of  const T&.
Or we want to put const & into the parameter when used.

Reply:
Unreal style guide is to call this TCallback too. Template things are called TSomething. If we do it without templating it then we would have to call them FSomethingCallback.